crates/ide-assists/src/handlers/generate_enum_projection_method.rs

```diff
@@ -156,15 +157,25 @@ fn generate_enum_projection_method(
         assist_description,
         target,
         |builder| {
-            let vis = parent_enum.visibility().map_or(String::new(), |v| format!("{v} "));
+            let vis = parent_enum.visibility().map_or(String::new(), |v| format!("{} ", v));
+
+            let field_type_syntax = field_type.syntax();
+
+            let must_use = if ctx.config.assist_emit_must_use {
+                "#[must_use]\n"
```

Comment:
```suggestion
                "#[must_use]\n    "
```
This need whitespace after, otherwise the indentation is messed up

Reply:
Added the same here, if the indentation was broken, tests would have failed :)

Reply:
Well, for one this is missing tests that actually check if the `must_use` insertion works. This will fail indentation when it inserts as the function will become
```rs
    #[must_use]
fn foo() ...
```
instead of
```rs
    #[must_use]
    fn foo() ...
```